### ISSUE TITLE
Valuechanges does nothing unless lowering them. + Language-strings in cfg useless.

### ISSUE THREAD

**Issue by Phasfeldt** [IndianaBonesUrMom/fivem-seatbelt#1]
So, I've been playing with the script a little and decided for my use, that launching out of the windshield with the low speed, while only hitting a small cardboard box on the sidewalk, isn't gonna be useful 😄 

So! I changed up the values, but going higher than 
```
Cfg.DiffTrigger = 0.549
Cfg.MinSpeed    = 9.25
```
renders the script useless, pretty much. No matter the speed, what you hit and the angle of the impact.
While lowering them, just makes it easier though.

Any clarification on it?

Also, the language-strings in the config is not used in the client.lua currently 😛

**Comment by JaroSound** [comment]
Yep sometimes when i drive 10 km/h i can fall... CFG does nothink...

**Comment by IndianaBonesUrMom** [comment]
Rising Cfg.MinSpeed doesn't do anything? lmao
#Imdone
#nofurthersupport
